MekHQ/src/mekhq/campaign/rating/CampaignOpsReputation.java

```diff
@@ -247,9 +247,6 @@ private int getTotalForceUnits() {
         int totalBattleArmor = 0;
         int forceTotal;
 
-        // Count total units for transport
-        getTotalCombatUnits();
```

Comment:
Do we know if each of the methods called by `getTotalCombatUnits` is side-effect free?

Reply:
They happen to be (just checked), but... nice catch!
I had deleted the call out of blind trust in findbugs and didn't think of checking all the methods called in `getTotalCombatUnits()` (which turn out to be simple getters, implemented only in `AbstractUnitRating`)